src/functions-and-arrays.js

```diff
@@ -29,8 +77,34 @@ const wordsUnique = [
   'bring'
 ];
 
+var uniquifyArray = function (arr) {
+  var arrFiltered = [];
+  for (var i = 0; i < arr.length; i++) {
+    if (arrFiltered.indexOf(arr[i]) === -1) {
+      arrFiltered.push(arr[i]);
+    }
+  }
+  return arrFiltered;
+};
+
+console.log(uniquifyArray(wordsUnique));
+
 // Iteration #6: Find elements
 const wordsFind = ['machine', 'subset', 'trouble', 'starting', 'matter', 'eating', 'truth', 'disobedience'];
+var doesWordExist = function (arr, word) {
+  for (let i = 0; i < arr.length; i++) {
```

Comment:
There's one piece of test you are not passing in jasmine. Maybe doing something like this you can go around it, although it may not be the best or cleanest way to do it:

var doesWordExist = function (arr, word) {
  for (let i = 0; i < arr.length; i++) {
    if (arr[i] === word) {
      return true;
    } 
  }
  return false
};

You could also use the .includes( ) method.